docs/source/configuration/reference.rst

```diff
@@ -502,7 +502,7 @@ ascii                       Theme without any unicode characters at all
         rendered. A lower number means that the segment has a higher priority.
 
         Segments are removed according to their priority, with low priority 
-        segments being removed first.
+        segments (i.e., with a greater priority number) being removed first.
```

Comment:
There are no examples of `i.e.,` in currently present documentation. (And, in fact, I almost never see comma there even though like some other Russian people I sometimes place comma far more often then needed because it is very common in our language.)